apple/OmnivoreKit/Sources/Models/DataModels/FeedItem.swift

```diff
@@ -31,6 +31,7 @@ public extension LinkedItem {
   var unwrappedSlug: String { slug ?? "" }
   var unwrappedTitle: String { title ?? "" }
   var unwrappedPageURLString: String { pageURLString ?? "" }
+  var unwrappedLocalPdfURL: String { localPdfURL ?? "" }
```

Comment:
Should probably remove this property since localPdfURL can actually be optional (the other ones here are just for properties that we know are there but get returned to us as optional because of Obj-C interop)